src/content/eth2/index.md

```diff
@@ -74,11 +74,13 @@ Eth2 is designed to scale without sacrificing performance, security or decentral
 
 ### Proof of stake {#proof-of-stake}
 
+Ethereum today is a [proof-of-work blockchain](/en/learn/#proof-of-work-and-mining). It relies on miners to keep the network secure and in sync by devoting a great amount of computing power to creating new blocks. Proof-of-stake keeps the network secure but replaces energy consumption with a financial commitment.
+
 Staking is the process of committing your ETH to become a validator for Ethereum. A validator runs software that confirms transactions and, when chosen, creates new blocks in the chain. To become a full validator you'll need 32 ETH. However there will be opportunities to offer a smaller stake by joining staking pools.
 
 There will be thousands of validators. And they will keep the network secure by processing transactions and making sure everyone agrees on the same data.
 
-Ethereum today is a [proof-of-work blockchain](/en/learn/#proof-of-work-and-mining). It relies on miners to keep the network secure and in sync by devoting a great amount of computing power to creating new blocks. Proof-of-stake keeps the network secure but replaces energy consumption with a financial commitment.
+Looking to learn more about staking? Find the [staking smart contract address](/eth2/staking-address/).
```

Comment:
@ryancreatescopy - FYI, I've added this link & copy to the main Eth2 page. I welcome your edits 😄